src/ripple/protocol/STBlob.h

```diff
@@ -179,7 +157,7 @@ class STBlob
     }
 
 private:
-    Blob value;
+    Buffer value;
```

Comment:
This is not your doing, but I'm a bit surprised that this member data does not have some kind of decorator.  `Buffer`, for example, uses a trailing `_` for its private member data.  So if we were following `Buffer`'s example this member would be named `value_`.